### ISSUE TITLE
CLB, Sectors page: missing repair tool for broken sectors 

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/checklistbank#1144]
It looks like, there is no tool to repair broken sectors through Project - Sectors page, if Rematch button does not fix the problem:

![image](https://user-images.githubusercontent.com/28574252/207628724-a058f75d-6ded-4e75-b804-65a078458086.png)

**Comment by mdoering** [comment]
I think this only exists in the dev branch for the sector tables. 
@thomasstjerne But I think you can go to the assembly tree and modify it manually there. Cumbersome, but for some sectors at least an interim solution.

**Comment by thomasstjerne** [comment]
Yes - this is tied to the dev branch where it is also possible to set priorities for sectors.
